aiohttp_pydantic/oas/docstring_parser.py

```diff
@@ -120,14 +120,28 @@ def tags(docstring: str) -> List[str]:
     return []
 
 
+def deprecated(docstring: str) -> bool:
+    """
+    Extract the "Deprecated:" block of the docstring.
+    """
+    iterator = LinesIterator(docstring)
+    for line in iterator:
+        if re.fullmatch("deprecated\\s*:.*", line, re.IGNORECASE):
+            iterator.rewind()
+            lines = " ".join(_i_extract_block(iterator))
+            if [" ".join(e.split()) for e in re.split("[,;]", lines.split(":")[1])][0] == "True":
```

Comment:
If you want match `Deprecated: True`  

```python
            lines = " ".join(_i_extract_block(iterator))
            return lines.split(":")[1].strip().lower() == 'true'
    return False
```

But for me it's better to match as true any text, and it's useful to keep the text inside docsting.

```
Deprecated:  This route is deprecated since ... use the route ... instead.
```

So deprecated should be:

``` python
def deprecated(docstring: str) -> bool:
    """
    Extract the "Deprecated:" block of the docstring.
    """
    iterator = LinesIterator(docstring)
    for line in iterator:
        if re.fullmatch("deprecated\\s*:.*", line, re.IGNORECASE):
            return True
    return False
```

And you can remove your modification in 
```
def operation(docstring: str
```
function.